### ISSUE TITLE
Dividends: enter dividends in different currency

### ISSUE THREAD

**Issue by chrisaut** [buchen/portfolio#1901]
**Is your feature request related to a problem? Please describe.**
It is useful to track dividend payments in the securities' original currency, no matter what currency the deposit accout or security price is tracked in.

For example, purchasing of a dividend paying US stock in a german exchange. Typically the security price will now be tracked in EUR, and the deposit account most likely also be in EUR. But dividends will be payed in USD (and converted on the fly into EUR).

This would also help to more clearly see dividend growths, independed of forex movements.

**Describe the solution you'd like**
During dividend entry, allow the user to select the dividend per share currency manually.

**Additional context**
Forum discussion: https://forum.portfolio-performance.info/t/jaehrliche-dividendensteigerung-auswerten/12589

**Comment by chirlu** [comment]
Duplicate of #1321. Related: #510.

**Comment by iwalucas** [comment]
I am doing this with internally having different deposit accounts... 
I have a DEGIRO EUR and a DEGIRO USD for example, so I keep things separated.

Not sure if that addresses your situation

**Comment by chrisaut** [comment]
> Duplicate of #1321. Related: #510.

Right, I should have linked those, but those are in German and AFAIK we try to do English now. Should I still close this?

**Comment by bbatsov** [comment]
I'd rather keep it open, as I don't know German myself and I wanted to request the same thing. :-) 

Like @iwalucas I'm handling this for the time being by doing a "fake" conversion of the dividend between my cash accounts, but that's pretty annoying as it doesn't reflect the fact that the dividend was actually paid in a different currency than the one the stock was bought in (e.g. you buy RDSB in Amsterdam in euros, but it pays dividends in GBP). I think a lot of people are buying on EU markets stocks that pay dividends in all sorts of different currencies, so that's a pretty common problem to have. 

I definitely think it'd be better if there was some way to specify the dividend currency of an instrument independently - e.g. by default you assume it's the same as the instrument's currency, but you can change it to something else.

**Comment by mcheaven** [comment]
> 
> 
> I am doing this with internally having different deposit accounts...
> I have a DEGIRO EUR and a DEGIRO USD for example, so I keep things separated.
> 
> Not sure if that addresses your situation

can you please go more into detail on this? Are you using the pdf import succesfully? Because that seems to only work with your degiro main currency but cant read dividends in other currencies.

**Comment by tomasherman** [comment]
This is an issue for me aswell. I could maybe take a stab at fixing it ... i assume one would need to:
- be able to specify dividend currency for given security (by default same one as the one that the security is denominated in)
- update the dividend dialog to use currency from the newly introduced currency, correct?

**Comment by chirlu** [comment]
Cf. #1915.

**Comment by bbatsov** [comment]
> be able to specify dividend currency for given security (by default same one as the one that the security is denominated in)

Such a solution would be fine by me, although it might be a bit of an overkill, as many securities don't pay dividends. In general I want to able to select some currency if the dividend dialog, but if I can also set a default somewhere else - that'd be great.

**Comment by tomasherman** [comment]
> > be able to specify dividend currency for given security (by default same one as the one that the security is denominated in)
> 
> Such a solution would be fine by me, although it might be a bit of an overkill, as many securities don't pay dividends. In general I want to able to select some currency if the dividend dialog, but if I can also set a default somewhere else - that'd be great.

Off topic: what a small world ... You don't know me but i love emacs prelude... Thanks! :-D

**Comment by bbatsov** [comment]
Happy to hear this! :-) Small world, indeed. If I didn't have so many OSS projects on my hands I would have probably implemented this feature myself as well. :D

**Comment by arjan-s** [comment]
Hi all, this issue has been open for nearly two years, and it seems there hasn't been any more work on this. This is still kind of preventing me to use Portfolio, as I have several ETFs that are denominated in EUR but pay out dividends in USD. I currently have to calculate by hand what the dividends in EUR are and enter everything manually. And this issue causes the (DEGIRO) PDF importer to fail too, as it doesn't understand the multiple currency per asset situation.

**Comment by mimijojo** [comment]
Also a problem for me.  Purchased security in EUR, getting dividends in USD.  Still trying to find a good workaround.
